src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/Ports.tsx

```diff
@@ -40,19 +44,25 @@ type Props = {
   className?: string
   name: string
   node: any
+  next?: sdk.NodeTransition[]
+  flowsName: any
 } & RouteComponentProps
 
-export class StandardPortWidgetDisconnected extends React.Component<Props> {
+export class StandardPortWidgetDisconnected extends React.PureComponent<Props> {
```

Comment:
You could convert this to a `FC` while you're at it

Reply:
Yup I think that would be nice for the future. There's a ton of possible optimization on the flow builder, but there's too much work testing & converting to do this in this PR